### PR TITLE
Update magda dev crawl sources

### PR DESCRIPTION
**Pull request by AlexGilleran**
### What this PR does
The dev.magda.io crawl sources have gotten a bit out of date with data.gov.au, so some data.gov.au bugs can't be replicated on it. This just takes recent changes to the data.gov.au crawl list and ports them across.

### CODE REVIEW

File: deploy/helm/magda-dev.yml [hunk 1]
```diff
@@ -115,73 +115,88 @@ minion-broken-link:
...
+    ## The service's image as per the global options under "global.image", but with a "name "
+    ## currently available are magda-ckan-connector, magda-project-open-data-connector and magda-csw-connector
+    - image:
+        name: "magda-ckan-connector"
+        # tag: vx.x.x
+        # pullPolicy: IfNotPresent
+        # imagePullSecret: ""
+
+      ## Unique id to identify this connector and records that are harvested from it
+      id: dga
```
Comment (t83714) [inline deploy/helm/magda-dev.yml]: connector with ID `dga` seems defined twice.
There is another one defined around line 196

**Comment by t83714** [comment]
Had another try in the cluster and seems BOM connector needs a new crawling url (getting 503 access denied error):
```
Failed to GET http://www.bom.gov.au/geonetwork/srv/eng/csw?service=CSW&version=2.0.2&request=GetRecords&constraintLanguage=FILTER&constraint_language_version=1.1.0&resultType=results&elementsetname=full&outputschema=http%3A%2F%2Fwww.isotc211.org%2F2005%2Fgmd&typeNames=gmd%3AMD_Metadata&startPosition=1&maxRecords=100. 1 retries left. Exception:
Error: Invalid Server Response or Empty result returned!
Requesting http://www.bom.gov.au/geonetwork/srv/eng/csw?service=CSW&version=2.0.2&request=GetRecords&constraintLanguage=FILTER&constraint_language_version=1.1.0&resultType=results&elementsetname=full&outputschema=http%3A%2F%2Fwww.isotc211.org%2F2005%2Fgmd&typeNames=gmd%3AMD_Metadata&startPosition=1&maxRecords=100
[xmldom warning]	unclosed xml attribute 
@#[line:6,col:144]
Error: Invalid Server Response or Empty result returned!
    at Request.request_1.default [as _callback] (/usr/src/app/component/dist/Csw.js:207:31)
    at Request.self.callback (/usr/src/app/node_modules/request/request.js:185:22)
    at emitTwo (events.js:126:13)
    at Request.emit (events.js:214:7)
    at Request.<anonymous> (/usr/src/app/node_modules/request/request.js:1161:10)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at IncomingMessage.<anonymous> (/usr/src/app/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:111:20)
```
But I guess it probably should be handled by a seperate issue. 
Do we want to remove it or leave it in for now? @AlexGilleran

**Comment by AlexGilleran** [comment]
Should probably raise an issue on magda-config-data.gov.au for it :)

**Comment by t83714** [comment]
Created a separate issue for BOM connector here:
https://github.com/magda-io/magda-config-data.gov.au/issues/30

**Pull request merged**